rust/benchmark-datasets/src/datasets/ms_marco_queries.rs

```diff
@@ -54,6 +55,8 @@ impl DocumentDataset for MicrosoftMarcoQueriesDataset {
         let buffered_reader = tokio::io::BufReader::new(file);
         let lines = LinesStream::new(buffered_reader.lines());
 
+        let mut unique_content_filter = BloomFilter::with_rate(0.01, 10_000_000);
```

Comment:
didn't want to use a HashSet here as that would defeat the purpose of streaming from the dataset